package.json

```diff
@@ -42,7 +42,7 @@
     "prepare": "husky install && npm-run-all clean rollup postinstall lib types webpack checkpoints",
     "rollup": "rollup -c",
     "test": "nyc mocha 'test/**/*.test.ts'",
-    "postinstall": "sh ./postinstall.sh",
```

Comment:
Pretty sure this fix is already on master.

Reply:
Ok will drop it

Reply:
I did some testing and install fails when using a github URL but if you install from npm tarball it works fine. I'm ok with omitting.